RPIPs/RPIP-49.md

```diff
@@ -15,39 +15,54 @@ This Info RPIP exists to explain an overall tokenomics proposal, the RPIPs it ha
 
 ## Components currently being advocated for
-1. [Bond curves](RPIP-42.md) - this is perhaps the most critical component, as it unlocks dramatically higher capital efficiency. Note that the ability to penalize at the node level is also specified in this RPIP.
-2. [Megapools](RPIP-43.md) - this is needed for the bond curve changes, otherwise gas would be prohibitive gas for, eg, 1.5-ETH bond minipools.
-3. [Forced exits](RPIP-44.md) - this is needed for the bond curve changes, to enable node level penalties to be effective.
-4. [RPL Burn](RPIP-45.md) - this is the primary value capture mechanism being proposed.
-5. [UVC](RPIP-46.md) - this is the mechanism that allows for splitting revenue between various targets (including voters to keep our governance robust). Note that it also has some thinking about how to find an appropriate share for NOs, as well as some future-looking settings that are not intended to be used immediately.
-6. [Remove operator-gated upgrades](RPIP-47.md) - this is desirable eventually. Since it reduces the number of things that need to be tested going forward, the thought is we should include it as quickly as possible.
-7. [Lower minipool requirements](RPIP-48.md) - this (a) enables a new size of minipool with 4 ETH bond from the NO and (b) removes the RPL staking requirement to create minipools. If not released alongside UVC, rETH commisison is changed to make these new pools only slightly more productive than the current 8-ETH minipools.
+1. [RPIP-42: Bond curves](RPIP-42.md) - this is perhaps the most critical component, as it unlocks dramatically higher capital efficiency. Note that this RPIP also includes (a) the ability to penalize at the node level and (b) the removal of the RPL staking requirement to create minipools.
+2. [RPIP-43: Megapools](RPIP-43.md) - this is needed for the bond curve changes, otherwise gas would be prohibitive gas for, eg, 1.5-ETH bond minipools.
+3. [RPIP-44: Forced exits](RPIP-44.md) - this is needed for the bond curve changes, to enable node level penalties to be effective.
+4. [RPIP-45: RPL Burn](RPIP-45.md) - this is the primary value capture mechanism being proposed.
+5. [RPIP-46: UVC](RPIP-46.md) - this is the mechanism that allows for splitting revenue between various targets (including voters to keep our governance robust). Note that it also has some thinking about how to find an appropriate share for NOs, as well as some future-looking settings that are not intended to be used immediately.
+6. [RPIP-47: Forced delegate upgrades](RPIP-47.md) - this is desirable eventually. Since it reduces the number of things that need to be tested going forward, the thought is we should include it as quickly as possible.
 
-1 requires 2 and 3.
-4 and 5 require each other.
+## Topics that need feedback, discussion, or fleshing out
+- [RPL Burn](RPIP-45.md) vs [RPL Buy & LP](RPIP-50.md)
+- 3-transaction deposits for greater queue efficiency
```

Comment:
What are 3-transaction deposits? Can you summarize or add a link to discussion of the concept?